pkg/config/config.go

```diff
@@ -45,7 +47,6 @@ type ClusterConfig struct {
 	ClusterCIDR          string `json:"clusterCIDR"`
 	ServiceCIDR          string `json:"serviceCIDR"`
 	ServiceNodePortRange string `json:"serviceNodePortRange"`
-	DNS                  string `json:"dns"`
```

Comment:
Would you be OK with leaving this field here and changing its tag to `json:"-"`? Then, assigning a value to it could happen as part of ReadAndValidate, and the callers don't need to call a new method or handle possible errors.

Reply:
I'm fine with that.